README.md

```diff
@@ -127,11 +174,30 @@ bundle exec rake foodrobot:send_weekly_summary
 
 ## Generating Sample Data
 
-Running the `db:sample_region` rake task will create a new `Region` in your database and populate it with a bunch of random data (volunteers, donors, recipients, schedule chains, etc.). For more info on what exactly is created, see `lib/sample_data/region_data.rb`.
+If you start with an empty schema, you'll want to start by creating a Volunteer user with the admin bit set, and a first region e.g.:
```

Comment:
Could we remove this bit? All this stuff should be set up by running `rake db:seed` right? Would folks realistically want to set this up manually like this without running the seeds?